src/plugins/intel_gpu/src/graph/graph_optimizer/post_input_reorder.cpp

```diff
@@ -65,7 +65,6 @@ void post_input_reorder::run(program& p) {
                                       layout_format,
                                       input_layout.data_padding);
                 auto& reorder = add_reorder(p, input, node, current_layout);
-                reorder.set_unique_id();
```

Comment:
Why is this removed? As we discussed offline, we'll continue to have unique id for various purpose in node.

Reply:
rollback to remove unique_id. thanks.